extensions/amp-analytics/0.1/vendors.js

```diff
@@ -100,174 +100,182 @@ import {NAVEGG_CONFIG} from './vendors/navegg';
+      'adobeanalytics': ADOBEANALYTICS_CONFIG,
+      'adobeanalytics_nativeConfig': ADOBEANALYTICS_NATIVECONFIG_CONFIG,
+      'afsanalytics': AFSANALYTICS_CONFIG,
+      'alexametrics': ALEXAMETRICS_CONFIG,
+      'amplitude': AMPLITUDE_CONFIG,
+      'atinternet': ATINTERNET_CONFIG,
+      'baiduanalytics': BAIDUANALYTICS_CONFIG,
+      'bg': BG_CONFIG,
+      'burt': BURT_CONFIG,
+      'byside': BYSIDE_CONFIG,
+      'chartbeat': CHARTBEAT_CONFIG,
+      'clicky': CLICKY_CONFIG,
+      'colanalytics': COLANALYTICS_CONFIG,
+      'comscore': COMSCORE_CONFIG,
+      'cxense': CXENSE_CONFIG,
+      'dynatrace': DYNATRACE_CONFIG,
+      'epica': EPICA_CONFIG,
+      'euleriananalytics': EULERIANANALYTICS_CONFIG,
+      'facebookpixel': FACEBOOKPIXEL_CONFIG,
+      'gemius': GEMIUS_CONFIG,
+      'googleadwords': GOOGLEADWORDS_CONFIG,
+      'googleanalytics': GOOGLEANALYTICS_CONFIG,
+      'gtag': GTAG_CONFIG,
+      'ibeatanalytics': IBEATANALYTICS_CONFIG,
+      'infonline': INFONLINE_CONFIG,
+      'iplabel': IPLABEL_CONFIG,
+      'keen': KEEN_CONFIG,
+      'kenshoo': KENSHOO_CONFIG,
+      'krux': KRUX_CONFIG,
+      'linkpulse': LINKPULSE_CONFIG,
+      'lotame': LOTAME_CONFIG,
+      'marinsoftware': MARINSOFTWARE_CONFIG,
+      'mediametrie': MEDIAMETRIE_CONFIG,
+      'mediarithmics': MEDIARITHMICS_CONFIG,
+      'mediator': MEDIATOR_CONFIG,
+      'metrika': METRIKA_CONFIG,
+      'moat': MOAT_CONFIG,
+      'mobify': MOBIFY_CONFIG,
+      'mparticle': MPARTICLE_CONFIG,
+      'mpulse': MPULSE_CONFIG,
+      'navegg': NAVEGG_CONFIG,
+      'newrelic': NEWRELIC_CONFIG,
+      'nielsen': NIELSEN_CONFIG,
+      'nielsen-marketing-cloud': NIELSEN_MARKETING_CLOUD_CONFIG,
+      'oewa': OEWA_CONFIG,
+      'oewadirect': OEWADIRECT_CONFIG,
+      'oracleInfinityAnalytics': ORACLEINFINITYANALYTICS_CONFIG,
+      'parsely': PARSELY_CONFIG,
+      'piStats': PISTATS_CONFIG,
+      'permutive': PERMUTIVE_CONFIG,
+      'piano': PIANO_CONFIG,
+      'pinpoll': PINPOLL_CONFIG,
+      'pressboard': PRESSBOARD_CONFIG,
+      'quantcast': QUANTCAST_CONFIG,
+      'retargetly': RETARGETLY_CONFIG,
+      'rakam': RAKAM_CONFIG,
+      'reppublika': REPPUBLIKA_CONFIG,
+      'segment': SEGMENT_CONFIG,
+      'shinystat': SHINYSTAT_CONFIG,
+      'simplereach': SIMPLEREACH_CONFIG,
+      'snowplow': SNOWPLOW_CONFIG,
+      'teaanalytics': TEAANALYTICS_CONFIG,
+      'tealiumcollect': TEALIUMCOLLECT_CONFIG,
+      'top100': TOP100_CONFIG,
+      'topmailru': TOPMAILRU_CONFIG,
+      'treasuredata': TREASUREDATA_CONFIG,
+      'umenganalytics': UMENGANALYTICS_CONFIG,
+      'upscore': UPSCORE_CONFIG,
+      'vponanalytics': VPONANALYTICS_CONFIG,
+      'webengage': WEBENGAGE_CONFIG,
+      'webtrekk': WEBTREKK_CONFIG,
+      'webtrekk_v2': WEBTREKK_V2_CONFIG,
+    });
 
-if (getMode().test || getMode().localDev) {
-  ANALYTICS_CONFIG['_fake_'] = _FAKE_;
-}
+// eslint-disable-next-line no-undef
+if (!ANALYTICS_VENDOR_SPLIT) {
```

Comment:
I'm good with this now. In the future, is it possible to build the fake json file for local testing, but never upload it to cdn?  Thanks.

Reply:
Yup, ill do that in another PR